crmsh/ui_context.py

```diff
@@ -255,6 +255,8 @@ def enter_level(self, level):
         self._in_transit = True
 
         entry = level()
+        if 'requires' in dir(entry) and not entry.requires():
+            self.fatal_error("Missing requirements")
```

Comment:
No, this will bring the issue https://github.com/ClusterLabs/crmsh/issues/1355 back, which was fixed in https://github.com/ClusterLabs/crmsh/pull/1300